Game/marvin.cpp

```diff
@@ -6,132 +6,130 @@
 #include "world/objects/npc.h"
 #include "camera.h"
 #include "gothic.h"
+#include "utils/stringutil.h"
+
+Marvin::Marvin(Gothic& gothic)
+: gothic( gothic )
+, commandHandlers()
```

Comment:
Nitpick: useless line

Reply:
You mean the commandHandlers init?
I am not sure if it is strictly necessary, but I once had the problem that I did not init a map in the constructor which lead to a crash while inserting elements.
Also I like to see in the constructor what the initial state of all the member variables are, this also avoids the constructor variable ordering warning.